rdr_service/dao/workbench_dao.py

```diff
@@ -510,7 +512,9 @@ def get_workspaces_with_user_detail(self, **kwargs):
             snapshot_subquery = (
                 session.query(func.max(WorkbenchWorkspaceSnapshot.id).label('snapshot_id'),
                               WorkbenchWorkspaceSnapshot.workspaceSourceId)
-                    .filter(WorkbenchWorkspaceSnapshot.excludeFromPublicDirectory == 0)
+                    .filter(or_(WorkbenchWorkspaceApproved.excludeFromPublicDirectory == 0,
+                                WorkbenchWorkspaceApproved.excludeFromPublicDirectory is None
```

Comment:
For the syntax, I think` WorkbenchWorkspaceApproved.excludeFromPublicDirectory.is_(None)` is the better way to do this.